module/zfs/vdev_raidz.c

```diff
@@ -1440,6 +1440,94 @@ vdev_raidz_child_done(zio_t *zio)
 	rc->rc_skipped = 0;
 }
 
+
+/* Rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_rotating_inc;
+extern int zfs_vdev_mirror_rotating_seek_inc;
+extern int zfs_vdev_mirror_rotating_seek_offset;
+
+/* Non-rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_non_rotating_inc;
+extern int zfs_vdev_mirror_non_rotating_seek_inc;
```

Comment:
The more appropriate style is to put `extern` variable declarations in a (new, if necessary) header to be included by both the mirror and raidz code. This means the compiler will complain in case anybody does anything funny like change the variable type in only one file.